mattermost/notices
# Gif picker deprecation notice

#### Summary
Gfycat deprecation notice as defined in [here](https://docs.google.com/document/d/1Hryg5tEGnEDhIHJuW-agreIw6Bb7Ghoh43IF7fb8oNA/edit#bookmark=id.8xeqmx3dc5yi)

#### Jira ticket
https://mattermost.atlassian.net/browse/MM-54280

#### Screenshots of the modals or screens in all target clients (required)

#### Test steps and expectation (required)

- Notice should appear for on-prem servers only: 
    - Servers below v7.8.9
    - v7.9-v7.10
    - v8.0.0 and v8.0.1
     - v8.1.0
- Notice should not appear in Cloud and should not appear in versions v.7.8.9, 8.0.2, 8.1.1 or 9.0 and above
- Notice should only appear for admins.
- Learn more button should go to the forum post.
